lang_php/analyze/ast_php.ml

```diff
@@ -233,6 +233,10 @@ and special =
   | This
   (* take many different forms in PHP, eval(), call_user_func, ${}, etc. *)
   | Eval
+  (* often transformed in Var "$self" in the analysis *)
```

Comment:
Could you also update a comment in the top of the file that says?
>  *  - some special keywords, for instance 'self', are transformed in
> *    "__special__self". See special() below.
The comment is still relevant but we should use a different example than `self`.